File: Dico.c
	//--------------------Compilation/lancement-----------//
		gcc leProgramme.c -o execuble.x
		./execuble.x
	//--------------------Types---------------------------//
		unsigned 
		signed
		auto	//instruction de création de variable(on force a crée)

		char

		short

		int	
		long
		long long

		float

		double

		*int pointeur 
		&addresse
	//--------------------Tableau et collection-----------//
		char* s = (char*) malloc (12*sizeof(char*));
		*s[0] = s ;

		struct aggrega
		{
			//aggrega de types
		};

		union aggrega2{
			//aggrega de types mais qui partage la meme memoire 
			//chacun peut avoir une valeur a la fois
		};

		enum enumeration {
			LUNDI,
			MARDI=2,
			MERCREDI,
			JEUDI,
			VENDREDI,
			SAMEDI,
			DIMANCHE;
		}
	//--------------------expression----------------------//

		|| ou
		&& et
		^  xou
		!= inegalité
		== egalité
		>= supOuEgal
		<= infOuEgal
	//--------------------if------------------------------//
		if (/* condition */)
		{
			/* code */
		}else{

		}

		((boolean)?val1:val2)

		switch(){
			case : break ;
			default : ;
		}
	//--------------------boucles-------------------------//
		while(){
			/* code */
		}

		do{
			/* code */
		}while();

		for (int i = 0; i < count; ++i)
		{
			/* code */
		}
	//--------------------function/sous programme---------//
		int function()
		{return 3;}

	//--------------------package-------------------------//
		//renommer le fichier en .h
		#import <stdio.h> ;
		extern
	//--------------------Exception-----------------------//

	//--------------------Multithread/task----------------//
		int pid=fork();
		wait(NULL);    //point de rendez vous

	//--------------------mot de language-----------------//

		
	break	
	case	
	const	
	continue
	default	
	do
	else	
	extern		//dans un autre fichier
	for	
	goto	
	if	
	register	//la variable va dans le CPU plutot que la memoire /!\ on ne peut acceder a l'addresse et elle doit etre local
	return
	sizeof	
	static		//variables persistante jusqu'a la fin du programme
	switch	
	volatile	//empeche l'optimisation du code de la variable
	while

File: Dico.java
	/*-------------------Compilation---------------------*/
		//javac Dico.java -encoding utf8
		//java Dico args
	

	/*-------------------Types---------------------------*/

		boolean bool = true ;

		byte   b = 1 ;
		int    i = 1 ;
		long   l = 3 ;
		float  f = 1 ;
		double d = 1 ;

		char   c = 'c';
		String s = "sss";

	/*--------------------Tableau et collection----------*/
		int[] tab  = new int[3] ;
		int[] tab2 = {1,2,3,4,5,6};

	/*-------------------expression----------------------*/
	/*
		||=ou
		&&=et
		^ =xou
		!= inegalité
		== egalité
		>= supOuEgal
		<= infOuEgal
		+ concatenation
	*/

	/*-------------------if-------------------------------*/

		if (bool) {}

		(bool?valeur1:valeur2)

		switch(){
			case '' -> ;
		}

	/*-------------------boucles-------------------------*/

		while (bool) {}
		do {} while (bool);
		while();

		for (int i=0 ;i<10 ;i++ ) {}
		for (objet t : iterableCollection) {}

	/*-------------------function/sous programme---------*/

		public static void main     (String[] args)   		{}
		public        void procedure(int param,int param2)	{}
		private       int  function ()						{ return 0 ;}

	/*-------------------package-------------------------*/
		package test;
		import  test;
	/*-------------------Exception-----------------------*/
		public void methode1() throws Exception ;
		throw new Exception ;

		try{

		}catch(Exception e){

		}finally{

		}
	/*--------------------Multithread/task---------------*/
		public class t extends Thread / implements Threadable{
			public void run()
			{}
		}

		
		synchronized //accès exclusif à un objet pendant l’exécution de la méthode
		//---> appel avec start();
	/*---------------------------------------------------*/
	/*                     Objet                         */
	/*---------------------------------------------------*/

		//constructeur
		public Dico(){} //----> appeller avec new

		//affichage
		public String toString(){}

		class classInterne{}

	/*-------------------heritage------------------------*/

		final class 			//classe qui ne peut etre deriver
		abstract class 			//classe qui doit etre heriter	

		extends Thread 			//class superieur
		implements Threadable 	//interface 

		abstract void methode1();
		default  void methode2();

		super. //methodes de la classe mere 
		this.  //celle ci 

	/*---------------------mot de language----------------*/

	abstract	
	continue	
	for	
	new	
	switch
	assert	
	default	
	goto	//ne pas utiliser
	package	
	synchronized
	do	
	if	
	private
	break	
	implements	
	protected	
	throw
	else	
	import	
	public	
	throws
	case	
	enum
	instanceof	
	return	
	transient
	catch	
	extends
	try
	final	
	interface	
	static
	class	
	finally
	strictfp
	volatile
	const	//ne pas utiliser
	native	
	super
	this
	while
